LanguageFeatures/Static-access-shorthand/type_inference_A01_t08.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Declaration denoted by a type scheme A context type scheme is
```

Comment:
It is probably more readable if the "local section title" is omitted:
```suggestion
/// @assertion A context type scheme is
```

Reply:
Similar cases in other tests.

Reply:
Updated for all tests

Reply:
Thanks!